### PR TITLE
[office-js, office-js-preview] Removing standard Outlook HTML tables

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

### CODE REVIEW

**Comment by AlexJerabek** [comment]
Needs sign off from @ElizabethSamuel-MSFT

**Comment by armanio123** [comment]
@AlexJerabek I'll wait for the review. Let me know when this is ready to be merged.

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -16557,11 +16521,14 @@ declare namespace Office {
...
-         *   <tr><td>Errors</td><td>Invalid date format - The date is not in an acceptable format.</td></tr>
-         * </table>
+         * 
+         * **{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}**: ReadWriteItem
+         * 
+         * **{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}**: Compose
+         * 
+         * **Errors**: 
+         * 
+         * - Invalid date format: The date is not in an acceptable format.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: Checked with the devs and "Invalid date format" is not an error. In this situation, they throw an exception. Let's remove this altogether.

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -14816,10 +14770,9 @@ declare namespace Office {
          *
          * @remarks
          *
-         * <table>
-         *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}</td><td>ReadMailbox</td></tr>
-         *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Compose or Read</td></tr>
-         * </table>
+         * **{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}**: ReadMailbox
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * **{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}**: ReadWriteMailbox
```

**Review by ElizabethSamuel-MSFT — CHANGES_REQUESTED** [review]

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. Updated accordingly.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Comment by AlexJerabek** [comment]
@armanio123 This has been reviewed and is good to go.

**Pull request merged**